script.js

```diff
@@ -0,0 +1,143 @@
+const newsWrapper = document.querySelector(".news-wrapper");
+const systemDate = document.querySelector(".system-date");
+
+let idList = Array.from({ length: 96 }, (_, idx) => idx);
+let isLightMode = true;
+let pageNum = 0;
+
+/**
+ * 배열을 섞는 함수
+ */
+const shuffleList = (list) => {
+  list.sort(() => Math.random() - 0.5);
+};
+
+/**
+ * 뉴스스탠드 Grid 제작하기
+ */
+const makeGrid = () => {
+  for (let i = 0; i < 24; i++) {
+    const gridItem = document.createElement("li");
+    const imgSrc = isLightMode
+      ? `./img/light-media/${idList[i]}.png`
+      : `./img/dark-media/${idList[i]}.png`;
+
+    let checkImg = new Image();
+    checkImg.src = imgSrc;
+    checkImg.onload = function () {
+      const img = document.createElement("img");
+      img.classList.add(`img${i}`);
+      img.src = imgSrc;
+      img.style.height = "20px";
+      gridItem.appendChild(img);
+    };
+
+    newsWrapper.append(gridItem);
+  }
+};
+
+/**
+ * 이미지 src 변경하기
+ */
+const changeImgSrc = () => {
+  let newImg = idList.slice(pageNum * 24, pageNum * 24 + 24);
+
+  for (let i = 0; i < 24; i++) {
+    const $img = document.querySelector(`.img${i}`);
+    const imgSrc = isLightMode
+      ? `./img/light-media/${newImg[i]}.png`
+      : `./img/dark-media/${newImg[i]}.png`;
+
+    let checkImg = new Image();
+    checkImg.src = imgSrc;
+    checkImg.onload = function () {
+      $img.src = imgSrc;
+    };
+    checkImg.onerror = function () {
+      $img.remove();
+    };
+  }
+};
+
+/**
+ * Grid 화살표 hidden 처리
+ */
+const setArrowVisible = (mediaList) => {
+  const leftArrow = document.querySelector(".left-arrow");
+  const rightArrow = document.querySelector(".right-arrow");
+
+  // 페이지 제한 0~3에 따른 hidden 여부
+  if (pageNum === 0) {
+    leftArrow.classList.add("hidden");
+  } else if (pageNum > 0 && pageNum < 3) {
+    leftArrow.classList.remove("hidden");
+    rightArrow.classList.remove("hidden");
+  } else if (pageNum === 3) {
+    rightArrow.classList.add("hidden");
+  }
+
+  // 언론사 로고 개수 따른 hidden 여부
+};
+
+/**
+ * Grid 화살표 클릭
+ */
+const clickArrow = (className) => {
+  if (className === "left-arrow") pageNum--;
+  if (className === "right-arrow") pageNum++;
```

Comment:
clicArrow의 인자를 -1 ,1로 받아서 pageNum += num으로 해도 될 것 같아요!!!!!!!
```suggestion
const clickArrow = (num) => {
    pageNum += num
```

Reply:
오 좋은 아이디어 감사합니다 - !! 👍👍